lib/Cake/Test/Case/Console/Command/Task/ViewTaskTest.php

```diff
@@ -233,14 +233,14 @@ public function testGetContent() {
 		);
 		$result = $this->Task->getContent('view', $vars);
 
-		$this->assertRegExp('/Delete Test View Model/', $result);
-		$this->assertRegExp('/Edit Test View Model/', $result);
-		$this->assertRegExp('/List Test View Models/', $result);
-		$this->assertRegExp('/New Test View Model/', $result);
```

Comment:
I see that there is a lot such changes. @tenkoma Do you think we could create `protected` method in `CakeTestCase` class to reuse assertion methods in old way? Thanks to that we could avoid changing all assertions.

But on the other hand: you already invested some time to fix that. Let's keep it.

Reply:
OK.

This replacement is indeed a lot.
However, it did not take much time because the editor's functions searched and replaced.

Reply:
OK, sounds fair. Resolving thread :)